docs/reference/indices/clone-index.asciidoc

```diff
@@ -49,20 +39,44 @@ PUT /my_source_index/_settings
   }
 }
 --------------------------------------------------
-// TEST[continued]
+// TEST[s/^/PUT my_source_index\n/]
 
 <1> Prevents write operations to this index while still allowing metadata
```

Comment:
In general, I think we should move toward more verbose intros to examples and fewer callouts. For example, here I'd introduce the examples with something like, "For example, the following request prevents write operations on `my_source_index` so it can be cloned. Metadata changes like deleting the index are still allowed."

Reply:
Addressed with 6b2b25d501a.